about/contributors.md

```diff
@@ -0,0 +1,25 @@
+## CloudEvents contributors
+
+* **Amazon**
+  * [AWS Lambda events](https://docs.aws.amazon.com/lambda/latest/dg/invoking-lambda-function.html)
+  * Arun Gupta, Ajay Nair, Rob Leidle, Orr Weinstein
+* **Google**
+  * [Google Cloud Functions](https://cloud.google.com/functions/) and [Cloud Functions for Firebase](https://firebase.google.com/docs/functions/)
+  * [Sarah Allen](https://github.com/ultrasaurus), [Rachel Myers](https://github.com/rachelmyers) (Serverless WG)
+  * [Thomas Bouldin](https://github.com/inlined), Morgan Hallmon, Robert-Jan Huijsman, Mike McDonald
+* **Microsoft**
+  * [Microsoft Event Grid](https://azure.microsoft.com/en-us/services/event-grid/)
+  * Bahram Banisadr, Clemens Vasters, Cesar Ruiz-Meraz, Dan Rosanova, Raja Ravipati
+* **Serverless Inc**
+  * [Serverless Framework and Event Gateway](https://serverless.com/)
+  * Austen Collins, Rupak Ganguly (Serverless WG)
+  * Brian Neisler, Jeremy Coffield, Ganesh Radhakirshnan
+* **IBM**
+  * [Doug Davis](https://github.com/duglin), [Daniel Krook](https://github.com/krook), Matt Rutkowski (Serverless WG)
```

Comment:
I guess if we're listing products please list IBM Cloud Functions: https://console.bluemix.net/openwhisk/